.gitignore

```diff
@@ -1,6 +1,10 @@
+# test files
+audio_test.mp3
+file_test.mat
+video_test.mp4
+
```

Comment:
i think it is better to use a folder and regex, rather than explicitly naming all the test files.